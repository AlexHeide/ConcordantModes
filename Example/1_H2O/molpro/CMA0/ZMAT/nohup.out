Initial Geometric Internal Coordinate Values:
['2', '1'] R1 = 1.8130525861320914
['3', '2'] R2 = 1.8130525861320914
['3', '2', '1'] A2 = 103.58217001794341
Final Geometric Internal Coordinate Values:
['2', '1'] R1 = 1.8130525861320914
['3', '2'] R2 = 1.8130525861320914
['3', '2', '1'] A2 = 103.58217001794341
Final - Initial Geometric Internal Coordinate Values:
R1 = 0.0
R2 = 0.0
A2 = 0.0
Normal Coordinate Values:
Normal Coordinate #1   :  1.81305
Normal Coordinate #2   :  1.81305
Normal Coordinate #3   :  1.80785
(3, 3, 9)
[[[0 -7.419263615012217e-06 -9.425120125680086e-06 0 7.419263615012217e-06 9.425120125680086e-06 0 0 0]
  [0.0 3.709600628260257e-06 4.712584603216441e-06 0.0 2.7740329022578336e-19 -9.425169206432882e-06 0.0 -3.7096006282605344e-06 4.712584603216441e-06]
  [0.0 -3.455278999356519e-06 -9.73777292305092e-06 0.0 8.560247698652377e-07 7.691720504440225e-06 0.0 2.5992542294912826e-06 2.0460524186106944e-06]]

 [[0.0 3.709600628260257e-06 4.712584603216441e-06 0.0 2.7740329022578336e-19 -9.425169206432882e-06 0.0 -3.7096006282605344e-06 4.712584603216441e-06]
  [0 0 0 0 -7.419263615011107e-06 9.425120125679532e-06 0 7.419263615011107e-06 -9.425120125679532e-06]
  [0.0 -2.599254229490866e-06 2.0460524186111107e-06 0.0 -8.560247698662084e-07 7.691720504439671e-06 0.0 3.4552789993570747e-06 -9.737772923050782e-06]]

 [[0.0 -3.455278999356519e-06 -9.73777292305092e-06 0.0 8.560247698652377e-07 7.691720504440225e-06 0.0 2.5992542294912826e-06 2.0460524186106944e-06]
  [0.0 -2.599254229490866e-06 2.0460524186111107e-06 0.0 -8.560247698662084e-07 7.691720504439671e-06 0.0 3.4552789993570747e-06 -9.737772923050782e-06]
  [0 2.166945941032894e-05 -1.7057643337449746e-05 0 0 3.411528667489949e-05 0 -2.166945941032894e-05 -1.7057643337449746e-05]]]
Traceback (most recent call last):
  File "/home/vulcan/mel64643/packages/concordantmodes/Example/1_H2O/molpro/CMA0/ZMAT/main.py", line 17, in <module>
    CMA_obj.run()
  File "/home/vulcan/mel64643/packages/concordantmodes/cma.py", line 66, in run
    s_vec.run(self.zmat_obj.cartesians_init, True, second_order=self.options.second_order)
  File "/home/vulcan/mel64643/packages/concordantmodes/s_vectors.py", line 478, in run
    raise RuntimeError
RuntimeError
