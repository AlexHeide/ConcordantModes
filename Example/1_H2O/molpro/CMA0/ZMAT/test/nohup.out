Initial Geometric Internal Coordinate Values:
['2', '1'] R1 = 1.8188612030765294
['3', '2'] R2 = 2.7467173588339957
['4', '3'] R3 = 1.8188612030765294
['3', '2', '1'] A2 = 99.64001839027763
['4', '3', '2'] A3 = 99.64001839027763
['4', '3', '2', '1'] D3 = 113.70002674961002
Final Geometric Internal Coordinate Values:
['2', '1'] R1 = 1.8188612030765294
['3', '2'] R2 = 2.7467173588339957
['4', '3'] R3 = 1.8188612030765294
['3', '2', '1'] A2 = 99.64001839027763
['4', '3', '2'] A3 = 99.64001839027763
['4', '3', '2', '1'] D3 = 113.70002674961002
Final - Initial Geometric Internal Coordinate Values:
R1 = 0.0
R2 = 0.0
R3 = 0.0
A2 = 0.0
A3 = 0.0
D3 = 0.0
self.s_2center_dict
{'B1': array([[-0.23439557,  0.80896112,  0.53911095],
       [ 0.23439557, -0.80896112, -0.53911095],
       [ 0.        ,  0.        , -0.        ],
       [ 0.        , -0.        ,  0.        ]]), 'B2': array([[-0.        ,  0.        ,  0.        ],
       [-0.99665078, -0.08177543, -0.        ],
       [ 0.99665078,  0.08177543,  0.        ],
       [ 0.        , -0.        ,  0.        ]]), 'B3': array([[-0.        ,  0.        ,  0.        ],
       [-0.        , -0.        , -0.        ],
       [-0.23439557,  0.80896112, -0.53911095],
       [ 0.23439557, -0.80896112,  0.53911095]])}
self.s_3center_dict
{'A1': array([[-0.53391225, -0.12114915, -0.05034532],
       [ 0.50898584,  0.42494373,  0.24943119],
       [ 0.02492642, -0.30379458, -0.19908587],
       [ 0.        , -0.        ,  0.        ]]), 'A2': array([[-0.        ,  0.        ,  0.        ],
       [-0.02492642,  0.30379458, -0.19908587],
       [-0.50898584, -0.42494373,  0.24943119],
       [ 0.53391225,  0.12114915, -0.05034532]])}
self.s_4center_dict
{'D1': array([[-0.02493756,  0.30393037, -0.46690408],
       [ 0.03046818, -0.37133566,  0.46690408],
       [-0.03046818,  0.37133566,  0.46690408],
       [ 0.02493756, -0.30393037, -0.46690408]])}
self.B
[[-0.23439557  0.80896112  0.53911095  0.23439557 -0.80896112 -0.53911095
   0.          0.         -0.          0.         -0.          0.        ]
 [-0.          0.          0.         -0.99665078 -0.08177543 -0.
   0.99665078  0.08177543  0.          0.         -0.          0.        ]
 [-0.          0.          0.         -0.         -0.         -0.
  -0.23439557  0.80896112 -0.53911095  0.23439557 -0.80896112  0.53911095]]
self.B
[[-0.23439557  0.80896112  0.53911095  0.23439557 -0.80896112 -0.53911095
   0.          0.         -0.          0.         -0.          0.        ]
 [-0.          0.          0.         -0.99665078 -0.08177543 -0.
   0.99665078  0.08177543  0.          0.         -0.          0.        ]
 [-0.          0.          0.         -0.         -0.         -0.
  -0.23439557  0.80896112 -0.53911095  0.23439557 -0.80896112  0.53911095]
 [-0.53391225 -0.12114915 -0.05034532  0.50898584  0.42494373  0.24943119
   0.02492642 -0.30379458 -0.19908587  0.         -0.          0.        ]
 [-0.          0.          0.         -0.02492642  0.30379458 -0.19908587
  -0.50898584 -0.42494373  0.24943119  0.53391225  0.12114915 -0.05034532]
 [-0.02493756  0.30393037 -0.46690408  0.03046818 -0.37133566  0.46690408
  -0.03046818  0.37133566  0.46690408  0.02493756 -0.30393037 -0.46690408]]
6
[5.44320583e-04 5.44320583e-04 5.44320583e-04 3.42971452e-05
 3.42971452e-05 3.42971452e-05 3.42971452e-05 3.42971452e-05
 3.42971452e-05 5.44320583e-04 5.44320583e-04 5.44320583e-04]
u
[[5.44320583e-04 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00]
 [0.00000000e+00 5.44320583e-04 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00]
 [0.00000000e+00 0.00000000e+00 5.44320583e-04 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00]
 [0.00000000e+00 0.00000000e+00 0.00000000e+00 3.42971452e-05
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00]
 [0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  3.42971452e-05 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00]
 [0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 3.42971452e-05 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00]
 [0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 3.42971452e-05 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00]
 [0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 3.42971452e-05
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00]
 [0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  3.42971452e-05 0.00000000e+00 0.00000000e+00 0.00000000e+00]
 [0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 5.44320583e-04 0.00000000e+00 0.00000000e+00]
 [0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 5.44320583e-04 0.00000000e+00]
 [0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 0.00000000e+00
  0.00000000e+00 0.00000000e+00 0.00000000e+00 5.44320583e-04]]
Gmat
[[ 5.78617728e-04 -5.74330999e-06  0.00000000e+00 -1.23102750e-05
  -4.94809293e-06  1.91462475e-06]
 [-5.74330999e-06  6.85942904e-05 -5.74330999e-06 -1.85901189e-05
  -1.85901189e-05  0.00000000e+00]
 [ 0.00000000e+00 -5.74330999e-06  5.78617728e-04 -4.94809293e-06
  -1.23102750e-05  1.91462475e-06]
 [-1.23102750e-05 -1.85901189e-05 -4.94809293e-06  1.86292354e-04
   4.57868159e-06 -7.96899912e-06]
 [-4.94809293e-06 -1.85901189e-05 -1.23102750e-05  4.57868159e-06
   1.86292354e-04 -7.96899912e-06]
 [ 1.91462475e-06  0.00000000e+00  1.91462475e-06 -7.96899912e-06
  -7.96899912e-06  3.63037602e-04]]
Normal Coordinate Values:
Normal Coordinate #1   :  1.81886
Normal Coordinate #2   :  2.74672
Normal Coordinate #3   :  1.81886
Normal Coordinate #4   :  1.73905
Normal Coordinate #5   :  1.73905
Normal Coordinate #6   :  1.98444
Traceback (most recent call last):
  File "/home/vulcan/nlk36701/github/concordantmodes/Example/1_H2O/molpro/CMA0/ZMAT/test/main.py", line 16, in <module>
    CMA_obj.run()
  File "/home/vulcan/nlk36701/github/concordantmodes/cma.py", line 135, in run
    dir_obj_init.run()
  File "/home/vulcan/nlk36701/github/concordantmodes/directory_tree.py", line 73, in run
    with open(self.template, "r") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'templateInit.dat'
